util/index.ts

```diff
@@ -307,8 +319,10 @@ const getDisputeChannel = async order => {
  * @param {*} user
  * @returns i18n context
  */
-const getUserI18nContext = async user => {
-  const language = user.language || 'en';
+const getUserI18nContext = async (user: UserDocument) => {
+  const language = user.lang || 'en';  // UserDocument doesn't have language attr
+  // I think language always be 'en'
+  // BTW I fixed this!
```

Comment:
@Mersho what is going on here? this comments is completely confusing for future developers when they work on this file